docker-compose.dev.yml

```diff
@@ -50,8 +50,10 @@ services:
   gateway:
     volumes:
       - ".secrets/public-key.pem:/secrets/public-key.pem"
+      - ".secrets/private-key.pem:/secrets/private-key.pem"
```

Comment:
These values are probably not needed anymore, right?

Reply:
Fixed